numpy/testing/_private/utils.py

```diff
@@ -1441,9 +1441,10 @@ def assert_allclose(actual, desired, rtol=1e-7, atol=0, equal_nan=True,
     Raises an AssertionError if two objects are not equal up to desired
     tolerance.
 
-    The test is equivalent to ``allclose(actual, desired, rtol, atol)``.
-    It compares the difference between `actual` and `desired` to
-    ``atol + rtol * abs(desired)``.
+    The test is equivalent to ``allclose(actual, desired, rtol, atol)`` (but
+    note that ``allclose`` has with ``rtol=1e-05`` and ``atol=1e-08`` different
+    default values). It compares the difference between `actual` and `desired`
+    to ``atol + rtol * abs(desired)``.
```

Comment:
No need to specify the default values here, so "note that ``allclose`` has different default values" is sufficient.

Reply:
Agree. When I made the commit I was undecided if I should mention them or not. In the end I did, but I am equally happy without.